tensorflow/contrib/tensorrt/python/__init__.py

```diff
@@ -19,7 +19,16 @@
 from __future__ import print_function
 
 # pylint: disable=unused-import,line-too-long
-from tensorflow.contrib.tensorrt.python.ops import trt_engine_op
-from tensorflow.contrib.tensorrt.python.trt_convert import calib_graph_to_infer_graph
-from tensorflow.contrib.tensorrt.python.trt_convert import create_inference_graph
+try:
+  from tensorflow.contrib.tensorrt.python.ops import trt_engine_op
+  from tensorflow.contrib.tensorrt.python.trt_convert import calib_graph_to_infer_graph
+  from tensorflow.contrib.tensorrt.python.trt_convert import create_inference_graph
+except:
+  no_trt_message = (
+      '**** Failed to initialize TensorRT. This is either because the TensorRT'
+      ' installation path is not in LD_LIBRARY_PATH, or because you do not have it'
+      ' installed. If not installed, please go to'
+      ' https://developer.nvidia.com/tensorrt to download and install'
+      ' TensorRT ****')
+  raise RuntimeError(no_trt_message)
```

Comment:
Sorry I think it'll be better to make this change in tensorrt/__init__.py, as we're guarding the whole tensorrt package. I suggested the wrong path and sorry about that.

Also, can we raise the original exception? Like:
```
except Exception as e:
  no_trt_message = ...
  print(no_trt_message)
  raise e
```
I'll pull in this PR to test meanwhile.

Reply:
Pulled and things work well except one linter error: we'll need to add `  # pylint: disable=g-import-not-at-top` at the end of the import.
Basically the file would look like:
```
try:
  from google3.third_party.tensorflow.contrib.tensorrt.python import *  # pylint: disable=g-import-not-at-top
except Exception as e:
  no_trt_message = ...
  print(no_trt_message)
  raise e
```

Reply:
updated